src/main/python/janusgraph_python/core/attribute/TextPredicate/Text.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from gremlin_python.process.traversal import P
+
+
+class Text(object):
+    """
+    Class implementing Gremlin Python's Predicate class, and extending it for JanusGraph based Text Predicates.
+    """
+
+    @staticmethod
+    def textContains(value):
+        """
+        Implements JanusGraph's textContains functionality
+
+        Args:
+            value (str):
+
+        Returns:
+            P
+        """
+
+        predicate = P("textContains", value)
+        return predicate
+
+    @staticmethod
+    def textContainsPrefix(value):
+        """
+        Implements JanusGraph's textContainsFuzzy functionality.
```

Comment:
How about simply copying the comments from JanusGraph's docs? That's what I did for my initial PR for JanusGraph.Net which looks like this for this predicate:
```cs
/// <summary>
///     Is true if (at least) one word inside the text string begins with the query string.
/// </summary>
/// <param name="query">The query to search.</param>
/// <returns>The text predicate.</returns>
public static P TextContainsPrefix(string query) => new P("textContainsPrefix", query);
```
[Source](https://github.com/JanusGraph/janusgraph-dotnet/pull/1/files#diff-c48759f38b9b4bd06e880fb4aeb3ea80)

JanusGraph's docs on these predicates can be found [here](https://docs.janusgraph.org/latest/search-predicates.html#_text_predicate).

Reply:
Thanks for pointing those out! I've added those to the comments. :-)